app/views/devise/registrations/show.html.erb

```diff
@@ -34,35 +32,18 @@
         <p class="description-text"><%= @user.favourite_sports %></p>
     <% end %>
   </div>
+  <div class="row">
```

Comment:
Shouldn't the if be outside of the `row`? same on all of the above, so i would ask the design team @hpi-swt2/team-dz-17

Reply:
have you contacted any of them directly? You know, many of the developers have dis-watched this repo because of the notification mess

Reply:
Yeah, I think it's better to place the `<%if ... end %>` outside of the row so that no empty row is rendered. Don't you end up with a bunch of whitespace this way? (Haven't checked out the branch myself so far.)

Reply:
This is not a problem of this branch, all attributes are wrapped like this.